modules/workbench/modules/monitoring/modules/alert_policies/assets/alert_policies/task_failure_audit_project.json

```diff
@@ -0,0 +1,33 @@
+{
+  "combiner": "OR",
+  "conditions": [
+    {
+      "conditionThreshold": {
+        "aggregations": [
+          {
+            "alignmentPeriod": "60s",
+            "crossSeriesReducer": "REDUCE_SUM",
+            "perSeriesAligner": "ALIGN_RATE"
+          }
+        ],
+        "filter": "metric.type=\"cloudtasks.googleapis.com/queue/task_attempt_count\" resource.type=\"cloud_tasks_queue\" metric.label.\"response_code\"!=\"ok\" resource.label.\"queue_id\"=\"auditProjectQueue\"",
+        "comparison": "COMPARISON_GT",
+        "duration": "60s",
+        "thresholdValue": 0.1,
```

Comment:
this value differs from all the others, because we expect queue processing to be slow.  See https://github.com/all-of-us/workbench/pull/5249